emulator.py

```diff
@@ -27,9 +28,16 @@ def __init__(self):
         self.number_of_samples = [] # can add any other quantities of interest here
         self.val_loss = []
         
-    def read_data_from_folder(self):
-        # TODO: Use np.loadtxt so that we can append to self.x and self.y for training --> will depend on formatting of input
-        pass
+    def read_data(self, file):
+        infile = open(file,'rb')
+        new_dict = pickle.load(infile)
+        infile.close()
+        
+        for n, i in enumerate(new_dict["input_data"]):
+            self.x.append(new_dict["input_data"].values[n][1:])
+            
+        for data in new_dict["output_data"]:
+            self.x.append(new_dict["input_data"].values[n][1:])
```

Comment:
Shouldn't it be `self.y.append(new_dict["output_data"].values[n][1:])` in this line? (replace `x`->`y` and `input`->`output`)

Reply:
Made the changes below.